test/JDBC/input/BABEL_4553.mix

```diff
@@ -0,0 +1,102 @@
+-- tsql
```

Comment:
Let us have tests to verify the JOIN conditions that we have modified to confirm that we were getting incorrect result set in some cases previously but with this change we are getting the correct results.

Reply:
Added test for sys.sp_columns_100_view, sys.spt_columns_view_managed, sys.sp_pkeys_view
For sys.sp_statistics_view, the catalog query is tricky to understand.